lib/scihist_digicoll/env.rb

```diff
@@ -179,6 +179,8 @@ def self.persistent_redis_connection!
       # it'll just get out of sync.
     }
 
+    define_key :force_ssl, default: Rails.env.production?
```

Comment:
Make it into a boolean, where `"true"` turns into `true`, like so:

    define_key :force_ssl, default: Rails.env.production?, system_env_transform: Kithe::ConfigBase::BOOLEAN_TRANSFORM

Reply:
Great - done.